libs/sparkpost-lib/src/main/java/com/sparkpost/resources/ResourceTransmissions.java

```diff
@@ -22,6 +24,16 @@
 
     public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientArray trans) throws SparkPostException {
 
+        return createTransmission(conn, numRcptErrors, trans);
+    }
+
+    public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientList trans) throws SparkPostException {
```

Comment:
I have added a comment about this into issue

2016-04-14 22:58 GMT+03:00 Chris Wilson <notifications@github.com>:

> In
> libs/sparkpost-lib/src/main/java/com/sparkpost/resources/ResourceTransmissions.java
> <https://github.com/SparkPost/java-sparkpost/pull/41#discussion_r59780860>
> :
>
> > @@ -22,6 +24,16 @@
> >
> >      public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientArray trans) throws SparkPostException {
> >
> > +        return createTransmission(conn, numRcptErrors, trans);
> > +    }
> > +
> > +    public static TransmissionCreateResponse create(RestConnection conn, Integer numRcptErrors, TransmissionWithRecipientList trans) throws SparkPostException {
>
> I think instead of two methods we could just use TransmissionBase. Do you
> think it is better to overload the method?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SparkPost/java-sparkpost/pull/41/files/7cd1ad9ec2d5275c9f32e83ab339a39340fafd50#r59780860>
>



-- 
С уважением, Кальник Александр.